src/main/java/net/rptools/maptool/model/library/addon/AddOnLibraryImporter.java

```diff
@@ -191,6 +197,25 @@ public AddOnLibrary importFromFile(File file) throws IOException {
     }
   }
 
+  private void addMetaData(
+      String namespace, ZipFile zip, Map<String, Pair<MD5Key, Type>> pathAssetMap)
+      throws IOException {
+    var entries = zip.stream().filter(e -> !e.getName().contains("/")).toList();
+    for (var entry : entries) {
+      String path = METADATA_DIR + entry.getName();
+      try (InputStream inputStream = zip.getInputStream(entry)) {
+        byte[] bytes = inputStream.readAllBytes();
+        MediaType mediaType = Asset.getMediaType(entry.getName(), bytes);
+        Asset asset =
+            Type.fromMediaType(mediaType)
+                .getFactory()
+                .apply(namespace + "/" + METADATA_DIR + path, bytes);
```

Comment:
Is it intentional to add `METADATA_DIR` to `path` a second time here?

Reply:
Would you believe me if I said yes? No? yeah I wouldn't either... 

I have updated so that the metadata dir doesn't get appended twice to the name of the asset